db/migrate/20161105203858_delete_geocoder_info_from_trip.rb

```diff
@@ -0,0 +1,7 @@
+class DeleteGeocoderInfoFromTrip < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :trips, :latitude
+    remove_column :trips, :longitude
```

Comment:
The current development schema does not include any of these columns. Maybe they were just on your local machine from when you were working with google maps? I'm worried that we will get an error if we try to merge this migration into development and it tries to delete columns that don't exist.